snorkel/labeling/model/label_model.py

```diff
@@ -622,6 +622,8 @@ def _execute_logging(self, loss: torch.Tensor) -> Metrics:
 
     def _set_logger(self) -> None:
         self.logger = Logger(self.train_config.log_freq)
+        if self.config.verbose:
+            logging.basicConfig(level=logging.INFO)
```

Comment:
This appears to be changing the default logging the logging module, not for this specific class's logger. Can you modify just the logging level of self.logger instead?

Reply:
I did consider this. But, the logging has been implemented throughout the code at the root level. The logging inside `fit()` of `LabelModel` and `log()` of `Logger` are using module level logging function `logging.info()`. Therefore all the logs are generated by root level logger and the only way to set log level for `LabelModel` specifically is to change the `CustomeLogger` significantly. However, I don't think that is necessary. Even if I change the root logger level to `INFO`, it would not interfere with the logging flow. Because, before creating an `INFO` level log, there will always be a check for the verbose parameter.

Reply:
I see. Alright, this looks good to me then!